charts/zeebe-benchmark/values.yaml

```diff
@@ -71,11 +73,23 @@ camunda-platform:
         configMap:
           name: zeebe-config
           defaultMode: 0754
+      - name: pyroscope
```

Comment:
:thinking: I think we don't need to mount the config map if we just use it for the env vars

Reply:
ah ok